util/release-odemis

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+# Release a new Odemis release
+# Beforehand, make sure you have already:
+#  1. switched to the right branch (eg, v2.6-branch)
+#  2. updated setup.py with the correct version number
+#  3. updated the changelog with dch -i
+#  4. all is commited
+# Call with "release-odemis"
+#
+# Note: for a new major release (eg v2.7), it does NOT create a new branch
+# Create it with:
+# git checkout -b v2.7-branch $commit-number-to-branch-from$
```

Comment:
I do not understand how step 1 and this note relate to each other. Step 1 seems to suggest that you should already be on the right branch, and the note seems to suggest that a new branch will be created.

Reply:
No, the note indicates that it does *NOT* create the branch. I just meant that there is nothing special for the "major" release.